### DISCUSSION TITLE
Tiempo de reproducción

### DISCUSSION THREAD

**Discussion by julianM2000** [fast4x/RiMusic#1005] [Ideas]
Por favor quitar el tiempo de la lista en general y solo dejar el tiempo restante que falta para que termine la música 
![Screenshot_20240310-121317](https://github.com/fast4x/RiMusic/assets/68936429/37fd2429-fc98-44b0-9975-ce67a5b55391)

**Comment by fast4x** [comment]
Hola, estoy trabajando en cambiarlo
